locallibrary/locallibrary/urls.py

```diff
@@ -15,21 +15,19 @@
 """
 from django.contrib import admin
 from django.urls import path
+from django.urls import include
```

Comment:
gom lại import 1 dòng thôi nếu cùng packed nha
```suggestion
from django.urls import path, include
```